src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -116,6 +117,21 @@
         (assoc acc project-name (.getInputStream jar-file jar-entry)))
       acc)))
 
+(defn get-stream-from-project-jar
+  [lein-project file-path]
+  (let [project-jar-file (fs/file (:target-path lein-project)
+                                  (str
+                                   (:name lein-project)
+                                   "-"
+                                   (:version lein-project)
+                                   ".jar"))]
+    (if (fs/exists? project-jar-file)
+      (let [project-jar (JarFile. project-jar-file)]
+        (if-let [project-jar-entry (find-file-in-jar project-jar
+                                                     file-path)]
+          (.getInputStream project-jar project-jar-entry)))
+      (log/warnf "Unable to find project jar file: '%s'" project-jar-file))))
```

Comment:
Maybe we should bail here with an exception / kill the ezbake run instead?  I wasn't sure if there were any "normal" circumstances where this could happen.